lib/danger/plugin_support/plugin_file_resolver.rb

```diff
@@ -17,26 +18,25 @@ def resolve_to_paths
       # When given a list of gems
       elsif @refs and @refs.kind_of? Array
         Bundler.with_clean_env do
-          Dir.mktmpdir do |dir|
-            gem_names = @refs
-            deps = gem_names.map { |name| Bundler::Dependency.new(name, ">= 0") }
+          # We don't use the block syntax as we want it to persist until the OS cleans it on reboot
+          # or whatever, it needs to persist outside this scope.
+          dir = Dir.mktmpdir
```

Comment:
in that case, you should store the dir somewhere and make sure to unlink it when you're done using it